internal/cmd/options.go

```diff
@@ -43,6 +45,15 @@ func (opts *LoggingFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 
 // SetLoggerLevel sets up the logger based on common options.
 func (opts *LoggingFlagOpts) SetLoggerLevel(ctx context.Context) context.Context {
+	logger := log.GetLogger(ctx)
+	ctx = executableTrace.WithExecutableTrace(ctx, &executableTrace.ExecutableTrace{
+		ExecuteStart: func(executableName, action string) {
+			logger.Info("started executing credential helper program %s with action %s", executableName, action)
```

Comment:
Should it be `Info` level or `debug` level? The process for accessing credential information is too detailed for a Notation user, and it is not the main logic of Notation.

Reply:
changed to `debug` level.